app/src/main/java/com/example/citadelpoc/Citadel.kt

```diff
@@ -11,7 +11,9 @@ class Citadel(context: Context) {
 
     public fun getBridgeToken(callback: (String?) -> Unit) {
         var endpoint = "bridge-tokens/"
-        VolleySingleton.getInstance(appContext).addToRequestQueue(Request.Method.POST, endpoint, null) { response ->
+        var body = JSONObject()
+        body.put("product_type", BuildConfig.citadelProductType)
```

Comment:
Let's add tracking_info here?

Reply:
We could create a simple generator of tracking info. to generate `my-comp-34ac4c4f-9622-4b4c-a186-1c8f90db93a2` tokenks for every bridge token.
We  could force semantic in tracking info( it will have a bit with troubleshooting on our side) and show hot trading info could be used